deploy/kube/README.md

```diff
@@ -14,7 +14,7 @@ mixer server with our test client (`mixc`), providing some attribute values:
 
 ```shell
 $ MIXS=$(minikube service mixer --url --format "{{.IP}}:{{.Port}}" | head -n 1)
-$ bazel run //cmd/client:mixc -- check -m $MIXS -a source.name=source,target.name=target,api.name=myapi,response.code=200,response.latency=100,client.id=$USER
+$ bazel run //cmd/client:mixc -- report "something happened" -m $MIXS -a source.name=source,target.name=target,api.name=myapi,api.method=v1.foo.bar,response.http.code=200,response.latency=100,client.id=$USER
```

Comment:
it doesn't matter at all, but I might make the example `report "http request"...` instead `report "something happened"` .